src/backend.rs

```diff
@@ -151,7 +151,11 @@ impl Backend {
             Event::Start(Tag::Paragraph),
             Event::Start(Tag::Image {
                 link_type: LinkType::Inline,
-                dest_url: rel_path.to_string_lossy().to_string().into(),
```

Comment:
is this definitely necessary?
markdown should be platform-independent. I don't believe backslashes are required in filepaths embedded in markdown, even in a windows environment

Reply:
> is this definitely necessary? markdown should be platform-independent. I don't believe backslashes are required in filepaths embedded in markdown, even in a windows environment

No they should be there, but somehow the backslash came in and in the html it was then escaped with some %HexNumber, will try to run the integration tests on Windows and paste the output

https://users.rust-lang.org/t/path-to-url-easiest-way/28632

Something like this might also work: `path.components().map(|c| urlencode(c.as_bytes())).join("/")`

Reply:
Running cargo test without and with my commit:
![image](https://github.com/user-attachments/assets/dcb73f8c-5831-4b03-a676-86910fd6b816)
Unfortunately temp file is gone but the test proves the expected string is not there ;)